app/react/src/server/framework-preset-cra.ts

```diff
@@ -3,7 +3,20 @@ import { Configuration } from 'webpack';
 import { logger } from '@storybook/node-logger';
 import { applyCRAWebpackConfig, getReactScriptsPath, isReactScriptsInstalled } from './cra-config';
 
+// Disable the built-in preset if the new preset is detected.
+const builtInPresetDisabled = (() => {
+  try {
+    require.resolve('@storybook/preset-create-react-app');
```

Comment:
More accurate if we check the user's `presets.js` file?

Reply:
We could, you would recommend just loading it and seeing what the export is?

Reply:
up to you, just wanted to raise it